### PR TITLE
add genericscheme callData raw data to the proposal details view

### PR DESCRIPTION
**Pull request by dkent600**
Fixes https://github.com/daostack/alchemy/issues/2021 for 2.0.

### CODE REVIEW

File: src/components/Proposal/ProposalSummary/ProposalSummaryNFTManager.tsx [hunk 1]
```diff
@@ -55,20 +56,24 @@ export default class ProposalSummaryNFTManager extends React.Component<IProps, n
...
-                <br/>
+                <br />
                 <div>
                   <strong>NFT Contract:</strong> <a href={linkToEtherScan(decodedCallData.values[1])} target="_blank" rel="noopener noreferrer">{decodedCallData.values[1]}</a>
                 </div>
                 <div>
                   <strong>TokenID:</strong> {decodedCallData.values[2]}
                 </div>
+                <div>
+                  <strong>Raw call data:</strong>
```
Comment (roienatan) [inline src/components/Proposal/ProposalSummary/ProposalSummaryNFTManager.tsx:74]: Don't we prefer to eliminate the use of `<strong>` and define the layout using div and CSS?

File: src/components/Proposal/ProposalSummary/ProposalSummaryCO2ken.tsx [hunk 2]
```diff
@@ -16,6 +17,14 @@ interface IProps {
 
 export default class ProposalSummaryCO2ken extends React.Component<IProps, null> {
 
+  private rawCallData(proposalState: IGenericPluginProposalState) {
+    return <>
+      <p>Raw call data:</p>
```
Comment (roienatan) [inline src/components/Proposal/ProposalSummary/ProposalSummaryCO2ken.tsx:22]: Please use `i18next` here and in the rest of the places.

File: src/components/Proposal/ProposalSummary/ProposalSummaryNFTManager.tsx (hunk 1 above)
Comment (dkent600) [reply src/components/Proposal/ProposalSummary/ProposalSummaryNFTManager.tsx:74]: I just didn't feel like refactoring the entire thing

File: src/components/Proposal/ProposalSummary/ProposalSummaryCO2ken.tsx (hunk 2 above)
Comment (dkent600) [reply src/components/Proposal/ProposalSummary/ProposalSummaryCO2ken.tsx:22]: Done

File: src/components/Proposal/ProposalSummary/ProposalSummaryNFTManager.tsx (hunk 1 above)
Comment (dkent600) [reply src/components/Proposal/ProposalSummary/ProposalSummaryNFTManager.tsx]: On the code that is coming in from outside contributors, we can hopefully get them to follow these standards before accepting their PRs.

**Pull request merged**